plugins/cloud-costs/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.2.2
```

Comment:
should we make this a minor bump just in case?

Reply:
We could but 0.2.1 has a bug - we would need to patch it as well, so not sure if it's worthy